process/elasticproc/logsevents/informativeLogsProcessor.go

```diff
@@ -31,23 +30,54 @@ func (ilp *informativeLogsProcessor) processEvent(args *argsProcessEvent) argOut
 
 	tx, ok := args.txs[args.txHashHexEncoded]
 	if !ok {
-		return argOutputProcessEvent{
-			processed: true,
-		}
+		return processEventNoTx(args)
 	}
 
 	switch identifier {
-	case writeLogOperation:
+	case core.CompletedTxEventIdentifier:
```

Comment:
incomplete check here. Please also add a SC deploy test. Reference: https://github.com/multiversx/mx-chain-proxy-go/blob/1cf28ef12cd859151a3c41d195b919bba3b4c43e/process/transactionProcessor.go#L465

Reply:
added an integration test for the sc deploy scenario